### ISSUE TITLE
Arguments do not work on the native .exe executable.

### ISSUE THREAD

**Issue by OskarZyg** [FabricMC/fabric-installer#71]
Using java to use the CLI version of the fabric installer works perfectly, however, this doesn't appear to work with the exe version. It just opens the GUI.
It would be useful to have this working, as when automating a fabric install for end users, they wouldn't need their `JAVA_PATH` set or bother with installing java in the first place.

**Comment by modmuss50** [comment]
Looks like an easy fix, I have opened an issue.

**Comment by modmuss50** [comment]
Going to reopen this as its not much use as the log output/status code is not forwarded from the jvm.

**Comment by OskarZyg** [comment]
Any progress?

**Comment by modmuss50** [comment]
I dont think this is something that really needs to be fixed imo. The windows exe version is mostly targeted towards normal players wanting to install. Using the jar when in the cli seems better to me.

**Comment by OskarZyg** [comment]
It's still a pretty nice feature to have, it's useful in some edge cases.
